<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QWidget" name="ChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatRoom</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTextBrowser" name="tb_recv"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="te_send"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pb_send">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
